src/web/src/components/Posts/Timeline.tsx

```diff
@@ -50,12 +49,17 @@ const Timeline = ({ pages, totalPosts, nextPage }: Props) => {
 
   // Iterate over all the pages (an array of arrays) and then convert all post
   // elements to <Post>
-  const postsTimeline = pages.map((page: Post[]): Array<ReactElement> | ReactElement =>
-    page.map(({ id, url }: Post) => {
-      currentCount += 1;
-
+  const postsTimeline = pages.map((page: Post[], idx1: number):
+    | Array<ReactElement>
+    | ReactElement =>
+    page.map(({ id, url }: Post, idx2: number) => {
       return (
-        <PostComponent postUrl={url} key={id} currentPost={currentCount} totalPosts={totalPosts} />
+        <PostComponent
+          postUrl={url}
+          key={id}
+          currentPost={idx1 * 5 + idx2 + 1}
```

Comment:
This magic number, `5`, is a problem.  We need to pass this down as a prop, since we could change how many items we return each time, and break this.  Probably we need to remember the old number of items, and subtract from the new number to get this offset.

Reply:
The amount of returned items is set on server, so I have no way of passing it down, is it correct?

Reply:
You know how many are returned each time, so use the `length` of the results.  It just might change in the future.